src/Resources/config/config.yml

```diff
@@ -8,6 +8,10 @@ knp_snappy:
     pdf:
         enabled: true
 
+twig:
+    globals:
+        invoiceLogoPath: "%kernel.project_dir%/vendor/sylius/invoicing-plugin/assets/sylius-logo.png"
```

Comment:
How can we make sure that it works? Why do we need another path for the test environment?

Reply:
Because in a plugin test application we don't have an invoice plugin in vendors and in a real application we do :/ I'm not pleased with that approach, but I don't see any other way to force KnpSnappyBundle to render the image that is inside the repository. 

The ideal scenario would be to check the PDF content in a Behat test - it's possible, afaik, but requires some work, definitely not for this PR. For now, I've just checked the new invoice template in a Sylius-Standard application and it worked with no problems 🚜

Reply:
Can we pass a relative path using a bundle notation? `@SyliusInvoicingPlugin/logo.png` and then use `Symfony\Component\HttpKernel\Config\FileLocator` to resolve it?

Reply:
I've tried it and it didn't work :| I see the images problem is one of the most common: https://github.com/KnpLabs/KnpSnappyBundle/issues/82, https://github.com/KnpLabs/KnpSnappyBundle/issues/232, https://github.com/KnpLabs/KnpSnappyBundle/issues/204, https://github.com/KnpLabs/snappy/blob/master/doc/faq.md#q-how-to-convert-page-with-relative-links-from-stdin--how-to-use-relative-media-urls

Reply:
Ok, apparently it works xD thanks @pamil